modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -402,6 +402,7 @@ const DomParser = (settings: DomParserSettings = {}, schema = Schema()): DomPars
 
     // Sanitize the content
     purify.sanitize(body, getPurifyConfig(defaultedSettings, mimeType));
+    purify.removed = [];
```

Comment:
This isn't performance related, but just makes sure we don't end up with a memory leak since DOMPurify retains the elements that are removed when sanitizing.